modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -50,16 +49,24 @@ spec:
       version: v1
       resource: deployments
 ----
-
-A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
-
++
+A service binding request is created and if the database service connection request is successful, the application is redeployed and the connection is established.
++
 .Binding connector
 image::odc-binding-connector.png[]
-
++
 [NOTE]
 ====
 You can also use the context menu by dragging the dangling arrow to add and create a binding connection to an operator-backed service.
 
 .Context menu to create binding connection
 image::odc_context_operator.png[]
 ====
+
+. In the navigation menu, click *Topology*. The spring-petclinic deployment in the Topology view includes an Open URL link to view its web page.
+
+. Click the *Open URL* link.
+
+You can now view the Spring PetClinic sample application remotely to confirm that the application is now connected to the database service and that the data has been successfully projected to the application from the Crunchy PostgreSQL database service.
+
+The Service Binding Operator has successfully created a working connection between the application and the database service.
```

Comment:
i don't see how this paragraph fits into the procedure module template:
https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

Reply:
Keeping it as is as the earlier content was an intended outcome instead of a particular verification section.